core/App/ehsm_provider.cpp

```diff
@@ -1013,6 +1021,18 @@ ehsm_status_t VerifyQuote(ehsm_data_t *quote,
     //set current time. This is only for sample use, please use trusted time in product.
     current_time = time(NULL);
 
+    memcpy(sgx_mr_enclave, (quote->data) + 64, 32);
+    memcpy(sgx_mr_signer, (quote->data) + 128, 32);
```

Comment:
you should convert the quote->data into sgx_quote3_t pointer, and then parse it to get the mrenclave/mrsigner

Reply:
you can put the logic into a separate static functon, 
e.g. 
get_mrenclave_from_quote()
get_mrsigner_from_quote()

Reply:
copy understood